pages/linux/wine.md

```diff
@@ -0,0 +1,32 @@
+# wine
+
+> Run Windows programs on Unix.
+> More information: <https://wiki.winehq.org/List_of_Commands>.
+
+- Run a program with args:
+
+`wine {{program}} {{program args}}`
+
+- Start a program:
```

Comment:
I guess the difference between start and run is that start spawns a process in the background ? We should clarify that, because otherwise it is not clear.

Reply:
I agree with your note. I will update the page accordingly.